llvm/lib/MCCAS/MCCASObjectV1.cpp

```diff
@@ -3124,6 +3135,7 @@ struct DIEVisitor {
   DataExtractor DistinctExtractor;
   DataExtractor::Cursor DistinctCursor;
   StringRef DistinctData;
+  uint8_t AddressSize;
```

Comment:
`DataExtractor`s can have their AddressSize set, maybe we are (or should be?) setting it already?